elliott/elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -474,6 +475,10 @@ def _verify_blocking_bugs(self, blocking_bugs_for, is_attached=False):
                 if is_attached and blocker.status in ['ON_QA', 'Verified', 'VERIFIED']:
                     blocker_advisories = blocker.all_advisory_ids()
                     if not blocker_advisories:
+                        # check if we need to release X+1 release this week
```

Comment:
Can this be checked to see if the blocking test should run at all, and not for every issue if it is not attached in the next version? 

Think the new release date check should decide if `blocker.all_advisory_ids()` should be run.

Reply:
good point, updated in https://github.com/openshift-eng/art-tools/pull/1066/commits/b63a6c14107cdf5a9bb6f716bb9258fc8ab2dc66